index.css

```diff
@@ -77,3 +78,75 @@
 .qt-add-btn:hover {
   background-color: #e0e0e0;
 }
+
+/* Custom Styles for radio/checkbox inputs */
+.quiz-item input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark--radio {
```

Comment:
use BEM for classnames or inner classes, ex: .quiz-item__checkmark--radio or .quiz-item .checkmark--radio
otherwise it might be sudenly overridden by project css files